serverless.yml

```
@@ -26,6 +26,7 @@ provider:
         - https://webappint.fromdoppler.net
         - https://webappqa.fromdoppler.net
         - http://localhost:3000
+        - http://localhost:35740
```

What do you think about also adding all the other report domains?

```suggestion
        - reports2.fromdoppler.com
        - reportsqa.fromdoppler.net:400
        - reportsint.fromdoppler.net:400
        - http://localhost:35740
```